.changeset/thirty-moose-allow.md

```diff
@@ -0,0 +1,6 @@
+---
+"gitbook-docs": minor
+"nextjs-website": minor
+---
+
+Add support for various swagger tags, so that they are shown
```

Comment:
The purpose of this activity is to make sure that all the `swagger` blocks in the documentation are properly parsed and rendered in the Developer Portal, even if the baseUrl attribute is empty and the documentation is hardcoded inside the block. You reach this objective by adding the "support for various swagger tags"